pkg/controller/gkenetworkparamset/gkenetworkparamset_controller.go

```diff
@@ -92,11 +98,33 @@ func (c *Controller) Run(numWorkers int, stopCh <-chan struct{}, controllerManag
 				c.queue.Add(key)
 			}
 		},
+		DeleteFunc: func(obj interface{}) {
```

Comment:
what are you doing for this event?

Reply:
https://github.com/kubernetes/cloud-provider-gcp/pull/530/files/e39803709f1494e102bb0ea25c71a9c8233ea9b1#diff-b87c941360d9d0eb1c44c79d31228e1b52d048d4369ead97e7c044e8722cc467R234

Reply:
per offline discussion, this will be handled via update event.

Reply:
done